test/test_dataloader.py

```diff
@@ -8,15 +8,26 @@
 import time
 import traceback
 import unittest
+import subprocess
 from torch import multiprocessing
 from torch.utils.data import Dataset, TensorDataset, DataLoader, ConcatDataset
 from torch.utils.data.dataset import random_split
-from torch.utils.data.dataloader import default_collate, ExceptionWrapper
+from torch.utils.data.dataloader import default_collate, ExceptionWrapper, MANAGER_STATUS_CHECK_INTERVAL
 from common import TestCase, run_tests, TEST_NUMPY, IS_WINDOWS
-from common_nn import TEST_CUDA
 
+TEST_CUDA = torch.cuda.is_available()
```

Comment:
If we import `TEST_CUDA` from `common_nn` multiple times during the execution of the test, `TEST_CUDNN = TEST_CUDA and torch.backends.cudnn.is_acceptable(torch.cuda.FloatTensor(1))` in `common_nn` will also be executed multiple times and CUDA on Windows doesn't seem to like it. This is an attempt to find a workaround for the issue.

Reply:
OK, can this live as a code comment, so someone doesn't "helpfully" refactor `TEST_CUDA` back into `common_nn`?